### PR TITLE
Cleanup usage of NodesToAddCollector, return nodes directly instead

### PR DESCRIPTION
**Pull request by TomasVotruba**
- rebuild docs
- fixup! rebuild docs
- return nodes directly

### CODE REVIEW

File: tests/Issues/AddNodeAfterNodeStmt/Source/AddNextNopRector.php [hunk 1]
```diff
@@ -51,13 +41,9 @@ public function refactor(Node $node)
             )
         );
 
-        $this->nodesToAddCollector->addNodeAfterNode(
+        return [
```
Comment (samsonasik) [inline tests/Issues/AddNodeAfterNodeStmt/Source/AddNextNopRector.php:44]: This test is on purpose for add node after node with inline html usages, so need to be reverted.

File: tests/Issues/AddNodeAfterNodeStmt/Source/AddNextStmtRector.php [hunk 2]
```diff
@@ -34,17 +28,9 @@ public function getNodeTypes(): array
 
     public function refactor(Node $node)
     {
-        if (isset($this->justAdded[$this->file->getFilePath()])) {
-            return null;
-        }
-
-        $this->nodesToAddCollector->addNodeAfterNode(
+        return [
```
Comment (samsonasik) [inline tests/Issues/AddNodeAfterNodeStmt/Source/AddNextStmtRector.php:31]: This test is on purpose for add node after node with inline html usages, so need to be reverted.

File: tests/Issues/AddNodeBeforeNodeStmt/Source/AddBeforeStmtRector.php [hunk 3]
```diff
@@ -34,17 +28,9 @@ public function getNodeTypes(): array
 
     public function refactor(Node $node)
     {
-        if (isset($this->justAdded[$this->file->getFilePath()])) {
-            return null;
-        }
-
-        $this->nodesToAddCollector->addNodeBeforeNode(
+        return [
```
Comment (samsonasik) [inline tests/Issues/AddNodeBeforeNodeStmt/Source/AddBeforeStmtRector.php:31]: This test is on purpose for add node after node with inline html usages, so need to be reverted.

Comment (TomasVotruba) [reply tests/Issues/AddNodeBeforeNodeStmt/Source/AddBeforeStmtRector.php:31]: It does the same, but now directly :+1:  The node adder will be removed, as not needed.

**Pull request merged**